shared/devices/container.js

```diff
@@ -4,12 +4,14 @@ import * as DevicesGen from '../actions/devices-gen'
 import * as ProvisionGen from '../actions/provision-gen'
 import * as RouteTree from '../actions/route-tree'
 import * as Constants from '../constants/devices'
+import * as I from 'immutable'
 import {compose, connect, setDisplayName, safeSubmitPerMount} from '../util/container'
 import {partition} from 'lodash-es'
 
 const mapStateToProps = state => ({
   _deviceMap: state.devices.deviceMap,
   waiting: Constants.isWaiting(state),
+  newlyChangedItemIds: state.devices.getIn(['isNew'], []),
```

Comment:
we actually don't want the fallback item here, else it'll make a new array each time which causes this method to thrash. also if its not a prop thats going directly through mergeProps we should prefix with an underscore.
```
_newlyChangedItemIds: state.devices.isNew
```

Reply:
👍

Reply:
actually put it back to the un-underscored because i thought it would be better to have the devices component tell the device rows whether they're new instead of having each device row figure it out.

Reply:
but the prop is not emitted from mergeProps, therefore its really only for mapStateToProps to pass to mergeProps, so it should use an underscore. Are you thinking of a different prop? or am i missing something

Reply:
cool